src/components/Pages/Classes/SinglePage/SingleClassView.js

```diff
@@ -63,9 +63,9 @@ function SingleClassView(props) {
     subject,
     link,
     finished
-  } = props.activeClasses;
+  } = props.singleClass;
 
-  console.log("activeClasses", props.activeClasses);
+  console.log("singleClass", props.SingleClass);
```

Comment:
Same here, I just don't see where it's imported.
Also above in lines 60-61, first name and last name columns have been changed to "name".

Reply:
Rectified. Should have been 'singleClass'